packages/blueprints-integration/src/timeline.ts

```diff
@@ -4,6 +4,13 @@ export { TSR }
 
 export { Timeline } from 'timeline-state-resolver-types'
 
+export enum TimelineObjClassesCore {
+	RundownRehearsal = 'rundown_rehersal',
+	RundownActive = 'rundown_active',
+	BeforeFirstPart = 'before_first_part',
+	IsLastPart = 'last_part',
```

Comment:
I would go with 'NoNextPart' instead of 'IsLastPart' as it makes it clearer that there could technically be more parts, but they arent playable. In general this will mean the end of the playlist, but not always